src/unwind.cc

```diff
@@ -36,11 +35,20 @@ void find_dso_add_error_frame(DDRes ddres, UnwindState *us) {
 }
 
 void add_container_id(Process &process, UnwindState *us) {
-  auto container_id = process.get_container_id();
+  const auto &container_id = process.get_container_id();
```

Comment:
This is slightly controversial, though the lifetime of the container ID should be tied to the process, which will outlive the aggregation of the sample